homeassistant/components/myuplink/manifest.json

```diff
@@ -0,0 +1,10 @@
+{
+  "domain": "myuplink",
+  "name": "myUplink",
+  "codeowners": ["@pajzo"],
+  "config_flow": true,
+  "dependencies": ["application_credentials"],
+  "documentation": "https://www.home-assistant.io/integrations/myuplink",
+  "iot_class": "cloud_polling",
+  "requirements": ["myuplink==0.0.6"]
```

Comment:
The library doesn't specify what requirements it needs. It looks like `aiohttp` is needed. Please add that either to `setup.cfg` or `pyproject.toml`.

Reply:
Still learning python - so where should i put `aiohttp` as requiment? In the `myuplink `package or...?

Reply:
Since you're using `setuptools` as build system in the library I recommend following their documentation on the subject:
https://setuptools.pypa.io/en/latest/userguide/pyproject_config.html

It's the `dependencies` setting that you want to use to specify library requirements.

Reply:
Fixed in version 0.0.9 of myuplink